app/src/main/java/org/oppia/android/app/home/recentlyplayed/RecentlyPlayedFragmentPresenter.kt

```diff
@@ -140,6 +99,55 @@ class RecentlyPlayedFragmentPresenter @Inject constructor(
     )
   }
 
+  private fun addRecentlyPlayedStoryListSection(
+    recentlyPlayedStoryList: MutableList<PromotedStory>
+  ) {
+    val recentSectionTitleViewModel =
+      SectionTitleViewModel(activity.getString(R.string.ongoing_story_last_week), false)
+    itemList.add(recentSectionTitleViewModel)
+    for (promotedStory in recentlyPlayedStoryList) {
+      val ongoingStoryViewModel = getOngoingStoryViewModel(promotedStory)
+      itemList.add(ongoingStoryViewModel)
+    }
+  }
+
+  private fun getOngoingStoryViewModel(promotedStory: PromotedStory): RecentlyPlayedItemViewModel {
+    return OngoingStoryViewModel(
+      promotedStory,
+      entityType,
+      fragment as OngoingStoryClickListener
+    )
+  }
+  private fun addOlderStoryListSection(olderPlayedStoryList: List<PromotedStory>) {
```

Comment:
Newline above the function for consistency.

Reply:
Done.